LoginPage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Formik, Field, Form } from 'formik';
+import { useHistory } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const LoginPage = () => {
+  const history = useHistory();
+
+  return (
+    <div className="container-fluid">
+      <div className="row">
+        <div className="col-md-6">
+          <img src="image.jpg" alt="Learning" style={{ height: '100vh', width: '100%' }} />
+        </div>
+        <div className="col-md-6">
+          <h2>Login</h2>
+          <p>Enter your email and password to access your account</p>
+          <Formik
+            initialValues={{ email: '', password: '' }}
+            onSubmit={(values, { setSubmitting }) => {
+              setTimeout(() => {
+                console.log('Logging in', values);
+                history.push('/dashboard');
+                setSubmitting(false);
+              }, 500);
+            }}
+          >
+            <Form>
```

Comment:
The 'Forgot Password' functionality is not implemented yet. It's better to disable the button or remove it until the functionality is ready. This would avoid confusing the users.